src/__tests__/components/CreateTemplate/ErrorMsg.test.tsx

```diff
@@ -0,0 +1,48 @@
+import ErrorMsg from "@/components/CreateTemplate/ErrorMsg";
+import { render, screen } from "@testing-library/react";
+import { INPUT_ERROR_MSG } from "@/constants/templateCreate";
+import { ITemplateErrorInput } from "@/interfaces/template";
+
+describe("ErrorMsg", () => {
+  it("should not display error message when no validations occur", () => {
+    const userInputError: ITemplateErrorInput = {
+      courtNameFullErr: false,
+      courtNameNullErr: false,
+      descriptionOverLimit: false,
+    };
+    const inputErrorMsg = INPUT_ERROR_MSG;
```

Comment:
You can use beforeEach to store all duplicated test cases in it. The last one of your test case only needs to modify a little and will make your test file cleaner if you want to.

Reply:
> You can use beforeEach to store all duplicated test cases in it. The last one of your test case only needs to modify a little and will make your test file cleaner if you want to.

beforeEach() is applied to initialised the test data before each test now